sql/ashamane/world/2018_07_13_99_world_nightbourne_spawn.sql

```diff
@@ -0,0 +1,556 @@
+(73151, 1, 100541, 0, 0, 0, 0, 0, 0, 0, 0), -- 73151
+(45565, 1, 2703, 0, 0, 13859, 0, 0, 0, 0, 0), -- 45565
+(133271, 1, 0, 0, 0, 0, 0, 0, 34268, 0, 0), -- 133271
+(12797, 1, 12403, 0, 0, 0, 0, 0, 0, 0, 0), -- 12797
+(45814, 1, 61337, 0, 0, 0, 0, 0, 50150, 0, 0), -- 45814
+(45563, 1, 2197, 0, 0, 13855, 0, 0, 0, 0, 0), -- 45563
+(133218, 1, 38196, 0, 0, 0, 0, 0, 0, 0, 0), -- 133218
+(3296, 1, 5289, 0, 0, 0, 0, 0, 2507, 0, 0), -- 3296
+(115736, 1, 132171, 0, 0, 0, 0, 0, 0, 0, 0), -- 115736
+(107126, 1, 28650, 0, 0, 0, 0, 0, 0, 0, 0), -- 107126
+(131413, 1, 133177, 0, 0, 0, 0, 0, 0, 0, 0), -- 131413
+(133185, 1, 10616, 0, 0, 0, 0, 0, 0, 0, 0), -- 133185
+(132029, 1, 132170, 0, 0, 0, 0, 0, 0, 0, 0), -- 132029
+(116658, 1, 140089, 0, 0, 0, 0, 0, 0, 0, 0), -- 116658
+(131421, 1, 132170, 0, 0, 0, 0, 0, 0, 0, 0), -- 131421
+(131425, 1, 133176, 0, 0, 0, 0, 0, 0, 0, 0), -- 131425
+(131414, 1, 133175, 0, 0, 0, 0, 0, 0, 0, 0), -- 131414
+(133186, 1, 109677, 0, 0, 0, 0, 0, 0, 0, 0), -- 133186
+(131326, 1, 132171, 0, 0, 0, 0, 0, 0, 0, 0), -- 131326
+(131420, 1, 133177, 0, 0, 137253, 0, 0, 0, 0, 0); -- 131420
+
+DELETE FROM creature_template_addon WHERE entry IN (133407);
+INSERT INTO `creature_template_addon` (`entry`, `path_id`, `mount`, `bytes1`, `bytes2`, `emote`, `aiAnimKit`, `movementAnimKit`, `meleeAnimKit`, `auras`) VALUES
+(133407, 0, 0, 0, 1, 0, 0, 0, 0, '262273');
+
+DELETE FROM creature_template WHERE entry IN (132026, 131326, 132027, 133186, 131420, 133695, 132032, 133407);
+INSERT INTO `creature_template` (`entry`, `KillCredit1`, `KillCredit2`, `modelid1`, `modelid2`, `modelid3`, `modelid4`, `name`, `femaleName`, `subname`, `TitleAlt`, `IconName`, `HealthScalingExpansion`, `RequiredExpansion`, `VignetteID`, `rank`, `family`, `type`, `type_flags`, `type_flags2`, `HealthModifier`, `ManaModifier`, `RacialLeader`, `movementId`, `VerifiedBuild`) VALUES
+(132026, 0, 0, 73853, 0, 0, 0, 'Chief Telemancer Oculeth', NULL, NULL, NULL, NULL, 6, 6, 0, 0, 0, 7, 0, 0, 1, 1, 0, 0, 26972), -- 132026
+(132027, 0, 0, 73854, 0, 0, 0, 'Arcanist Valtrois', NULL, NULL, NULL, NULL, 6, 0, 0, 1, 0, 7, 0, 0, 1, 1, 0, 0, 26972), -- 132027
+(133186, 0, 0, 82769, 82770, 82773, 83248, 'Leyline Navigator', NULL, NULL, NULL, NULL, 6, 0, 0, 0, 0, 7, 64, 0, 1, 1, 0, 123, 26972), -- 133186
+(131326, 0, 0, 67345, 0, 0, 0, 'First Arcanist Thalyssra', NULL, NULL, NULL, NULL, 6, 0, 0, 1, 0, 7, 0, 32768, 50, 1, 0, 0, 26972), -- 131326
+(131420, 0, 0, 83032, 83010, 0, 0, 'Duskwatch Sentinel', NULL, NULL, NULL, NULL, 6, 0, 0, 1, 0, 7, 2097224, 0, 17.4, 1, 0, 90, 26972), -- 131420
+(133695, 0, 0, 72841, 0, 0, 0, 'Maribeth', NULL, 'Innkeeper', NULL, NULL, 6, 6, 0, 0, 0, 7, 134217728, 0, 1, 1, 0, 0, 26972), -- 133695
+(132032, 0, 0, 70210, 0, 0, 0, 'Ly''leth Lunastre', NULL, NULL, NULL, NULL, 6, 0, 0, 0, 0, 7, 0, 0, 1, 1, 0, 0, 26972), -- 132032
+(133407, 0, 0, 82896, 0, 0, 0, 'Ambassador Blackguard', NULL, NULL, NULL, NULL, -1, 0, 0, 0, 0, 7, 0, 32768, 5, 1, 0, 0, 26972); -- 133407
```

Comment:
I would recommend to set Blackguard minlevel and maxlevel to 0. Since he have HealthScalingExpansion set to -1 (which already adds him Expansion maxlevel + level set on Minlevel/Maxlevel. So with the 0 he will have 110/120 when BfA comes)